### PR TITLE
disable meta upgrade from v1 to v3 directly

### PR DESCRIPTION
**Pull request by darionyaphet**
#### What type of PR is this?
- [ ] bug
- [ ] feature
- [ ] enhancement

#### What problem(s) does this PR solve?
Issue(s) number: 

Description:

#### Checklist:
Tests:
- [ ] Unit test(positive and negative cases)
- [ ] Function test
- [ ] Performance test
- [ ] N/A

Affects:
- [ ] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

#### Release notes:

Please confirm whether to be reflected in release notes and how to describe:
> ex. Fixed the bug .....

### CODE REVIEW

**Comment by darionyaphet** [comment]
Step 1: Set up nebula v1 meta service storage service and graph service. use console to show the hosts.

```
==============================================================================================
| Ip          | Port  | Status | Leader count | Leader distribution | Partition distribution |
==============================================================================================
| 192.168.8.5 | 55540 | online | 0            | No valid partition  | No valid partition     |
----------------------------------------------------------------------------------------------
| Total       |       |        | 0            |                     |                        |
----------------------------------------------------------------------------------------------
Got 2 rows (Time spent: 31.129/33.111 ms)

Tue Jan 25 11:57:39 2022
```

Step 2: Create space and tag

Step 3: shutdown the cluster and restart with nebula v3‘s meta service try to upgrade. The new meta service setup failed and could get info from the log.

```
I20220125 11:58:52.084648 58939 MetaDaemonInit.cpp:143] Get meta version is 1
E20220125 11:58:52.084671 58939 MetaDaemonInit.cpp:148] Can't upgrade meta from V1 to V3
```

**Review by critical27 — APPROVED** [review]

File: src/daemons/MetaDaemonInit.cpp [hunk 1]
```diff
@@ -145,13 +145,8 @@ std::unique_ptr<nebula::kvstore::KVStore> initKV(std::vector<nebula::HostAddr> p
...
-    auto ret = nebula::meta::MetaVersionMan::updateMetaV1ToV2(engine);
-    if (!ret.ok()) {
-      LOG(ERROR) << "Update meta from V1 to V2 failed " << ret;
-      return nullptr;
-    }
-
-    nebula::meta::MetaVersionMan::setMetaVersionToKV(engine, nebula::meta::MetaVersion::V2);
+    LOG(ERROR) << "Can't upgrade meta from V1 to V3";
+    return nullptr;
   } else if (version == nebula::meta::MetaVersion::V2) {
```
Comment (critical27) [inline src/daemons/MetaDaemonInit.cpp:150]: Why still exists `V2`, only V1 or V3 is return from `getMetaVersionFromKV`. So where do we call `updateMetaV2ToV3`?

**Review by critical27 — CHANGES_REQUESTED** [review]
Don't merge

File: src/daemons/MetaDaemonInit.cpp (hunk 1 above)
Comment (critical27) [reply src/daemons/MetaDaemonInit.cpp:150]: @HarrisChu , in previous test, does meta upgrade successfully? Something must be wrong here, either code or test

Comment (darionyaphet) [reply src/daemons/MetaDaemonInit.cpp:150]: if  kMetaVersionKey is existed, version will fetch from kv store

**Review by critical27 — APPROVED** [review]

File: src/daemons/MetaDaemonInit.cpp (hunk 1 above)
Comment (critical27) [reply src/daemons/MetaDaemonInit.cpp:150]: Never mind, my bad

**Review by panda-sheep — APPROVED** [review]
Good job

**Pull request merged**